src/sage/schemes/elliptic_curves/weierstrass_morphism.py

```diff
@@ -973,6 +973,105 @@ def inseparable_degree(self):
         """
         return Integer(1)
 
+    def is_identity(self):
+        r"""
+        Check if this Weierstrass isomorphism is the identity.
+
+        EXAMPLES::
+
+            sage: from sage.schemes.elliptic_curves.weierstrass_morphism import WeierstrassIsomorphism
+            sage: p = 97
+            sage: Fp = GF(p)
+            sage: E = EllipticCurve(Fp, [1, 28])
+            sage: ws = WeierstrassIsomorphism(E, None, E)
```

Comment:
```suggestion
            sage: ws = E.isomorphism_to(E); ws
            # add the output here
```
I think this will make it more clear, since it might be *slightly* confusing why the isomorphism from $E$ to $E$ is not the identity.

Reply:
This I don't get. My purpose was to show that it could happen

Reply:
Ohh right, nevermind then